src/CabanaPD_Force.hpp

```diff
@@ -624,19 +622,20 @@ class Force<ExecutionSpace, ForceModel<LPS, Fracture>>
                                                                   i );
             for ( std::size_t n = 0; n < num_neighbors; n++ )
             {
-                if ( mu( i, n ) > 0 )
-                {
-                    std::size_t j =
-                        Cabana::NeighborList<NeighListType>::getNeighbor(
-                            neigh_list, i, n );
-
-                    // Get the bond distance, displacement, and stretch.
-                    double xi, r, s;
-                    getDistance( x, u, i, j, xi, r, s );
-                    double theta_i =
-                        model.influence_function( xi ) * s * xi * xi * vol( j );
+                std::size_t j =
+                    Cabana::NeighborList<NeighListType>::getNeighbor(
+                        neigh_list, i, n );
+
+                // Get the bond distance, displacement, and stretch.
+                double xi, r, s;
+                getDistance( x, u, i, j, xi, r, s );
+                // Added mu to account for bond breaking.
+                double theta_i = mu( i, n ) * model.influence_function( xi ) *
+                                 s * xi * xi * vol( j );
+                // m( i ) is 0 when all bonds connected to particle i are
```

Comment:
"Check if all bonds are broken (m = 0) to avoid divide by zero. Equivalent to checking if this bond mu(i,n) is broken."